src/bp-groups/bp-groups-filters.php

```diff
@@ -215,29 +211,6 @@ function bp_groups_disable_at_mention_forums_notification_for_non_public_groups(
 	return $send;
 }
 
-/**
- * Use the mystery group avatar for groups.
- *
- * @since BuddyPress 2.6.0
- *
- * @param string $avatar Current avatar src.
- * @param array  $params Avatar params.
- * @return string
- */
-function bp_groups_default_avatar( $avatar, $params ) {
-	if ( isset( $params['object'] ) && 'group' === $params['object'] ) {
-		if ( isset( $params['type'] ) && 'thumb' === $params['type'] ) {
-			$file = 'mystery-group-50.png';
-		} else {
-			$file = 'mystery-group.png';
-		}
-
-		$avatar = buddypress()->plugin_url . "bp-core/images/$file";
-	}
-
-	return $avatar;
-}
-
```

Comment:
This function should go to the deprecated file.

Reply:
Done.